### PR TITLE
Add support for koa newer version

### PR DESCRIPTION
**Pull request by kevinrambaud**
Just found your package that is really awesome. As I'm often using koa for my project, I updated the code base to support newer version of Koa with async/await. It's a breaking change update.

It includes :

* replaced usage of generator functions by async/await
* updated examples
* updated readme
* updated dependencies
* updated package information and bumped package version from `0.3.2` to `1.0.0`

Let me know if I missed anything.

### CODE REVIEW

**Comment by kevinrambaud** [comment]
hey @nemtsov a moment to review it ?

File: README.md [hunk 1]
```diff
@@ -1,65 +1,69 @@
...
+Use it directly without this middleware if you need more flexibility._
 
 ## Installation
 
 ```
 $ npm install koa-json-mask
 ```
 
-## Options
+In order to use this plugin with the previous version of Koa using generator functions, please use version `0.3.2`
```
Comment (nemtsov) [inline README.md:26]: 👍

**Review by nemtsov — APPROVED** [review]
Oh, wow. This is perfect. Thank you so much for taking the time to contribute! Releasing it right away.

**Comment by nemtsov** [comment]
Released `koa-json-mask@1.0.0`

**Comment by nemtsov** [comment]
Awesome work, @kevinrambaud! 🎉

**Comment by kevinrambaud** [comment]
Great ! thanks a lot @nemtsov :)

**Pull request merged**